src/test/java/com/jcabi/manifests/ManifestsTest.java

```diff
@@ -127,4 +130,70 @@ public void appendsAttributesFromInputStream() throws Exception {
         );
     }
 
+    /**
+     * When get on multiple manifests, the first element takes precedence.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void getGetsTheFirstEntryAcrossMultipleManifests() throws Exception {
+        final String name = "Attr";
+        final Manifests mfs = new Manifests();
+        mfs.append(manifestStream("Attr: a\n"));
+        mfs.append(manifestStream("Attr: b\n"));
+        MatcherAssert.assertThat(
+            "The value is from the first manifest",
+            mfs.containsKey(name) && "a".equals(mfs.get(name))
```

Comment:
@westonized we should check for list equality here against the ordered list `a,b`